### PR TITLE
Prettify members page

### PR DESCRIPTION
**Pull request by staltz**
Will have to rebase this on `master` but I'll do that after dev mode is back (I need it for quickly developing with tailwind). So this PR is in draft mode.

Screenshot from 2021-02-22 17-47-47

### CODE REVIEW

File: web/handlers/admin/allow_list.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ func (h allowListH) overview(rw http.ResponseWriter, req *http.Request) (interfa
 	if err != nil {
 		return nil, err
 	}
+	// Reverse the slice to provide recent-to-oldest results
+	for i, j := 0, len(lst)-1; i < j; i, j = i+1, j-1 {
+		lst[i], lst[j] = lst[j], lst[i]
+	}
```
Comment (staltz) [inline web/handlers/admin/allow_list.go:70]: @cryptix What do you think about this?

Comment (cryptix) [reply web/handlers/admin/allow_list.go:70]: yup, nice use of the multi assign.

If you would want to sort by something other then the index of the array, I'd say go for [sort.Reverse](https://golang.org/pkg/sort/#Reverse) but this is fine until we change `List(ctx)` to have options and let the database do the work.

Comment (staltz) [reply web/handlers/admin/allow_list.go:70]: Oh, didn't know about `sort.Reverse`. Yeah, and I'll revisit this code once I work on pagination

Comment (cryptix) [reply web/handlers/admin/allow_list.go:70]: you would have to implement sort.Interface, like [here](https://github.com/cryptoscope/ssb/blob/f8b517c21b5d9b8fb05f923f29e59dda9a1730e5/sbot/status.go#L66).

**Review by cryptix — APPROVED** [review]

**Comment by staltz** [comment]
Thanks for reviewing! I'll rebase tomorrow. Signing-off from work for today :)

**Comment by staltz** [comment]
@cryptix This is ready, I added one commit to format the HTML. Updating to Go 1.16 was pretty straightforward.

One thing I noticed is that links such as

```html
<a href="javascript:history.back()" class="btn btn-primary">Back</a>
```

stopped working after the (I guess) CSP PR. Do you know how/why?

**Comment by staltz** [comment]
And... I'll work on fixing the tests. I should have ran them locally.

**Comment by cryptix** [comment]
> Do you know how/why?

Yup, you guessed right. That is inline code which is outlawed with the default CSP we adopted.

ps: i can shed some light on alternatives: If you really want inline content wit CSP, the options boil down to _nonces_ (which you need to add to the tag and set the nonce in the http header) or _hash the content of the tags_, which both requires quite the complex templating system setup to pull of. The third option is _unsafe_ which disables the protection all together.

**Comment by cryptix** [comment]
PS: on the tests, see what makes sense.

I just tried to show some examples of how we can test the content of the pages with goquery.

**Comment by cryptix** [comment]
on the back button specifically, I'd say we want to adopt #23 together with putting the _back redirect url_ in the template.

Then we don't have to involve page JS at all.

**Comment by staltz** [comment]
> on the back button specifically, I'd say we want to fix #23 together with putting the _back redirect url_ in the template.
> 
> Then we don't have to involve page JS at all.

I'm happy with this option, and I'm okay if the back button is broken in the meanwhile. There's anyway the browser back button.

PS: fixed the tests

**Comment by staltz** [comment]
Also, the HTML format is a bit strange, but it's a fine balance:

```html
<span
  id="welcome"
  class="text-center"
>{{i18n "AdminAllowListRemoveConfirmWelcome"}}</span>
```

There can't be spaces and newlines in the "content" portion because it would affect tests, and sometimes affects even layout.

**Comment by cryptix** [comment]
> There's anyway the browser back button.

yup. It's mostly a QOL thing, subjecting people to the obnoxious _do the POST again?!..._ warning.

**Comment by staltz** [comment]
Let's merge this?

**Comment by cryptix** [comment]
Sorry, was half way out for launch already.. :sweat_smile:

**Pull request merged**